### PR TITLE
SDIT-1398: ⬆️ Upgrade spring boot and minor fixes

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: rename-project.bash [hunk 1]
```diff
@@ -60,16 +60,19 @@ mv "src/main/${BASE}/hmppstemplatepackagename" "src/main/$BASE/$PACKAGE_NAME"
...
   helm_deploy/$PROJECT_NAME/values.yaml
 
-# Update helm values files with correct slack channels.
+# Update helm values files with correct slack channels.
 sed -i -z -E \
   -e "s/NON_PROD_ALERTS_SEVERITY_LABEL/$NON_PROD_ALERTS_SEVERITY_LABEL/" \
   helm_deploy/values-dev.yaml helm_deploy/values-preprod.yaml
 
+sed -i -z -E \
+  -e "s/PROD_ALERTS_SEVERITY_LABEL/$PROD_ALERTS_SEVERITY_LABEL/" helm_deploy/values-prod.yaml
```
Comment (petergphillips) [inline rename-project.bash:74]: I found out that this was missing when attempting to run the action

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/HmppsTemplateKotlin.kt [hunk 2]
```diff
@@ -3,7 +3,7 @@ package uk.gov.justice.digital.hmpps.hmppstemplatepackagename
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
 
-@SpringBootApplication()
+@SpringBootApplication
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/HmppsTemplateKotlin.kt:6]: unnecessary brackets

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/health/VersionOutputter.kt [hunk 3]
```diff
@@ -15,7 +15,7 @@ class VersionOutputter(buildProperties: BuildProperties) {
     log.info("Version {} started", version)
   }
 
-  companion object {
+  private companion object {
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/health/VersionOutputter.kt:18]: no need to be public

File: src/main/resources/logback-spring.xml [hunk 4]
```diff
@@ -12,8 +12,7 @@
     </encoder>
   </appender>
 
-  <logger name="uk.gov.justice.digital.hmpps.hmppstemplatepackagename.HmppsTemplateKotlinKt" additivity="false"
-          level="DEBUG">
+  <logger name="uk.gov.justice.digital.hmpps.hmppstemplatepackagename.HmppsTemplateKotlinKt" additivity="false" level="DEBUG">
```
Comment (petergphillips) [inline src/main/resources/logback-spring.xml:15]: annoying wrapping

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**